tools/scripts/check-packages-for-tslib.mjs

```diff
@@ -0,0 +1,38 @@
+import { getPackages } from '@manypkg/get-packages';
```

Comment:
```suggestion
/**
 * We've set `"importHelpers": true,` in `tsconfig.base.json`, meaning that when
 * `@csnx/npm-package` bundles our NPM libraries, anything that uses `tslib`
 * will import it from `tslib`, rather than inline it into the bundle.
 *
 * This helps consumers keep their bundles smaller, but it means that we need to
 * make sure that all of our libraries have (a valid) `tslib` as a peer
 * dependency.
 *
 * This script makes sure they do.
 */

import { getPackages } from '@manypkg/get-packages';
```